Makefile.deps

```diff
@@ -0,0 +1,25 @@
+YARNFLAGS :=
+YARN_ENVIRONMENT :=
+CARGOFLAGS :=
+
+ifeq ($(package),)
+	YARNFLAGS := ${YARNFLAGS} -A
+else
+	YARNFLAGS := ${YARNFLAGS} @hoprnet/${package}
+endif
+
+ifneq ($(origin PRODUCTION),undefined)
+	YARNFLAGS := ${YARNFLAGS} --production
+	YARN_ENVIRONMENT := ${YARN_ENVIRONMENT} DEBUG=
+endif
+
+ifneq ($(origin CI),undefined)
```

Comment:
Please document the used switch and reason.

Reply:
changed